APIs/UpcomingChanges.qmd

```diff
@@ -0,0 +1,100 @@
+# Upcoming changes
+
+This is the list of upcoming changes to Catalog APIs.
+
+## OpenSearch interface error handling update
+
+Please be informed that the **OData API** interface will be updated soon. The exact date of the update will be announced in a separate release note no later than 5 days before the scheduled change.
+
+To be compliant with the OData PRIP interfaces the API response for a single product query will be updated. The response will now return the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below.
+
+### OData request example
+
+::: {.panel-tabset}
+
+# HTTP Request
+[`https://datahub.creodias.eu/odata/v1/Products(f5439b10-deb4-420e-9b84-146035fbb16c)`](<https://datahub.creodias.eu/odata/v1/Products(f5439b10-deb4-420e-9b84-146035fbb16c)>){target="_blank"}
+
+:::
+
+### OData API response change
+
+#### Current response
```

Comment:
What do you think about the idea to keep current and new response side by side for easy comparision?

Reply:
I think this is a good idea, but I'm not sure how to change it.

Reply:
Alan, I tested two ways: 
1. With a layout that provides columns side by side it didn't look good and also looks like some more fixing would be needed. (If you prefer this we can further look into this in future)
2. Displaying two panel-tabset with two different content, seems to be much better. 

I pushed the second suggestion in your PR. Please let me know if you think it further needs improvement or if you prefer the old version.

Reply:
Thanks!